### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by kittisakppeam** [ChiWeiHsiao/DeepVO_PyTorch#60]
I trained model.Training loss is reduce very epcoh but validation loss isn't reduce following the training loss.Please reply me and thank you.

**Comment by kittisakppeam** [comment]
This is my result. I trained 100 epochs.It is overfitting and I evaluate the test sequence.

![Image](https://github.com/user-attachments/assets/650f62e0-c932-46a7-8f88-a3aad68ccc10)

Another sequence in valid set is the same.
